### DISCUSSION TITLE
When is the approximate cutoff date for commits to be included in the next preview version?

### DISCUSSION THREAD

**Discussion by danielchalmers** [dotnet/runtime#84317] [Q&A] [answered]
#82560 was merged Feb 23. Would it end up in preview 3 or later in preview 4? Is there a general rule for this? Thank you!

**Comment by huoyaoyuan** [comment, marked as answer]
If the dedicated branch is created for the next preview, then merged PRs after that won't get into the preview automatically and will need backporting. The version of`main` branch will be bumped to the next preview. You can check the version of `main` branch at `eng/Versions.props`, and the version of commit that PR get merged.
Usually the branch is created 1-2 weeks before the publish of a preview.
